api/cloud/v1/message.proto

```
@@ -1313,15 +1313,22 @@ message DatadogQueryRequest {
   int32 max_results = 8;      // limit returned items (0 = API default)
 
   int32 timeout_seconds = 9;
+
+  // JSON body for write operations (create-monitor, send-event, mute-monitor).
+  // Ignored for read queries.
+  string json_body = 10;
 }
 
 enum DatadogQueryType {
   DATADOG_QUERY_UNSPECIFIED = 0;
```

Buf (1.66.1)

[error] Enum value name "DATADOG_QUERY_UNSPECIFIED" should be prefixed with "DATADOG_QUERY_TYPE_".

(ENUM_VALUE_PREFIX)